app/services/internal/remove_stale_contacts_service.rb

```diff
@@ -0,0 +1,43 @@
+class Internal::RemoveStaleContactsService
+  def perform
```

Comment:
Pass an account_id here, so that we can limit the query to an account.

Reply:
make sense